src/main/java/com/sublinks/sublinksapi/api/lemmy/v3/authentication/config/SecurityConfig.java

```diff
@@ -33,14 +34,12 @@ public class SecurityConfig {
   public SecurityFilterChain filterChain(final HttpSecurity http) throws Exception {
 
     http
-        .authorizeHttpRequests((requests) -> requests
-            .anyRequest().permitAll()
-        )
-        .sessionManagement(
-            (sessionManagement) -> sessionManagement.sessionCreationPolicy(
-                SessionCreationPolicy.STATELESS)
-        )
-        .addFilterBefore(jwtFilter, UsernamePasswordAuthenticationFilter.class);
+        .csrf(AbstractHttpConfigurer::disable)
```

Comment:
Pretty sure this will trigger a security issue

Reply:
As thousand times said we can NOT enable it until we have another fix for lemmy uis csrf issue. See matrix chat.